contracts/eden/src/eden-micro-chain.cpp

```diff
@@ -1853,19 +1860,12 @@ void call(void (*f)(const action_context&, Args...),
    std::apply([&](auto&&... args) { f(context, std::move(args)...); }, t);
 }
 
-void remove_expired_inductions(const subchain::eosio_block& block)
+void remove_expired_inductions(const eosio::time_point& block_time, const status& status)
 {
-   auto& idx = db.status.get<by_id>();
-   if (idx.size() < 1)
-      return;  // skip if genesis is not complete
-
-   const auto& status = get_status();
-   if (!status.status.active)
-      return;  // skip if not active
+   if (status.migrationIndex < 5)
```

Comment:
Can we please not use a magic number?  You can derive the correct index from the variant, as in `migrations::is_completed`